<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Done</class>
 <widget class="QWidget" name="Done">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>You did it!</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>161</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>You did it!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Color Emoji</family>
     <pointsize>72</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ðŸŽ‰</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="clbReturn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
